rskj-core/src/test/java/org/ethereum/net/rlpx/HandshakeHandlerTest.java

```diff
@@ -92,12 +100,19 @@ void shouldDisconnectIfNoCapabilityIsPresent() throws Exception {
     }
 
     @Test
-    void shouldDisconnectIfRskCapabilityIsMissing() throws Exception {
+    void shouldDisconnectIfRskOrSnapCapabilityAreMissing() throws Exception {
```

Comment:
I think the title of this test shouldn't change, as we want to test if the RSK client is not connecting without RSK capability. 

Now as we have another real capability you can use the SNAP capability aswell to verify it.

Reply:
solved